### PR TITLE
add all components in one native exports

### PR DESCRIPTION
**Pull request by LucaMele**
add index js export for all components

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
+/* eslint-disable */
+import _AXACheckbox from './components/a-checkbox';
```
Comment (AndyOGo) [inline src/index.js]: You can export it directly by:

```js
export { default as AXACheckbox } from './components/a-checkbox';
```

https://stackoverflow.com/questions/35665759/es6-how-can-you-export-an-imported-module-in-a-single-line
https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export#Module_Redirects

Comment (LucaMele) [reply src/index.js]: nice thanks

Comment (LucaMele) [reply src/index.js]: will do asap

File: src/js/react-exports.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
+// First import the withReact integration helper function
```
Comment (AndyOGo) [inline src/js/react-exports.js]: don't forget to rebase onto develop

**Comment by LucaMele** [comment]
@AndyOGo PR FB implemented :)

**Pull request merged**